src/general/base_stage/StageBase.cs

```diff
@@ -38,6 +38,12 @@ public abstract class StageBase : NodeWithInput, IStageBase, IGodotEarlyNodeReso
 
     private bool transitionFinished;
 
+    /// <summary>
+    ///   Used to control how often light level is updated by the day/night cycle.
```

Comment:
```suggestion
    ///   Used to control how often updated light level data is read from the day/night cycle.
```
I think wording it like this will be more accurate to what is actually going on.